### ISSUE TITLE
Place the input in a form outside the list

### ISSUE THREAD

**Issue by telmiger** [tgrosinger/tw5-checklist#35]
For usability/accessibility reasons* I suggest to 
- put the input element for new list items in a form.
- include a submit button in the form (for users of touch devices and screenreaders)

This form should be placed outside the task list as it is not a task like the others.
